site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -495,7 +495,7 @@ components:
             "page":
                $ref: '#/components/schemas/PagingOptions'
             "filters":
-               $ref: '#/components/schemas/FilterOptions'
+               $ref: '#/components/schemas/FilterOptionsSet'
```

Comment:
are these ANDed together? In any case, we should document the semantics explicitly ...

Reply:
I updated the description. Let me know what you think